src/datadogstatsd.rs

```diff
@@ -0,0 +1,33 @@
+use dogstatsd::{Client, Options};
+
+pub struct  DdMetrics<'a>{
+    name: &'a str,
```

Comment:
If you want to avoid defining the lifetime here, we can make `name` be a `String` and copy the value in the `new` fn. Passing `String` by reference will be compatible with `&str` in subsequent calls.